<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelTransformerWidget</class>
 <widget class="QWidget" name="ModelTransformerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>128</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>254</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Model Transformer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Rotation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_rotateLeft">
       <property name="text">
        <string>-</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
       <property name="autoRepeatInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_rotateRight">
       <property name="text">
        <string>+</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
       <property name="autoRepeatInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_angle">
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>45.000000000000000</double>
       </property>
       <property name="value">
        <double>90.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_scaleAxis">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Scale Axis</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
       <property name="autoRepeatInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_scale">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Scale</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
       <property name="autoRepeatInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_scaleStep">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_xAxis">
       <property name="text">
        <string>X</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_yAxis">
       <property name="text">
        <string>Y</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_zAxis">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_setIdentity">
     <property name="text">
      <string>Set Identity</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
